static/locales/en-US.yaml

```diff
@@ -118,6 +118,7 @@ Search Filters:
   Fetching results. Please wait: Fetching results. Please wait
   Fetch more results: Fetch more results
   There are no more results for this search: There are no more results for this search
+  Clear Filters: Clear filters
```

Comment:
I think most of the places use capitalized words
e.g. settings page
![image](https://github.com/user-attachments/assets/42b19d6a-4d68-4f89-a689-8e86cc7a7910)

```suggestion
  Clear Filters: Clear Filters
```